src/Icon/README.md

```diff
@@ -65,4 +65,4 @@ const App = () => (
 * clock
 * up-down
 * upload-email
-* warning
\ No newline at end of file
+* warning
```

Comment:
Just checking that you meant to do the changes in this file as they look unrelated? If so ignore me.

Reply:
yeah I've just updated the comment since it was a copy paste from the Box component so I've updated it